tests/post_training/data/wc_reference_data.yaml

```diff
@@ -31,6 +35,10 @@ tinyllama_scale_estimation_per_channel_backend_OV:
   metric_value: 0.81389
   num_int4: 188
   num_int8: 124
+tinyllama_scale_estimation_per_channel_backend_TORCH:
```

Comment:
What about adding test for groupped quantization, since in most cases we use group_size=128 or 64?

Reply:
You can copy this configuration with AWQ and remove `awq: True` for now. After adding AWQ for Torch this test would be identical for OV and Torch.
```python
{
        "reported_name": "tinyllama_data_aware_awq_scale_estimation",
        "model_id": "tinyllama/tinyllama-1.1b-step-50k-105b",
        "pipeline_cls": LMWeightCompression,
        "compression_params": {
            "group_size": 64,
            "ratio": 0.8,
            "mode": CompressWeightsMode.INT4_SYM,
            "awq": True,
            "scale_estimation": True,
            "advanced_parameters": AdvancedCompressionParameters(
                scale_estimation_params=AdvancedScaleEstimationParameters(32, 5, 10, 1.0)
            ),
        },
```

Reply:
ptq - 296
performance - 40

Reply:
The results of the jobs:
Model | Backend | Metric name | Metric value | Metric diff | Num int4 | Num int8 | Compr. time (from Performance job) | RAM MiB (from Performance job)
-- | -- | -- | -- | -- | -- | -- | -- | --
tinyllama_scale_estimation_group_size_64 | OV | Similarity | 0.8566 | -0.1434 | 94 | 124 | 0:04:17 | 8681
tinyllama_scale_estimation_group_size_64 | TORCH | Similarity | 0.8566 | -0.1434 | 94 | 124 | 0:04:01 | 5575